airflow/jobs/local_task_job.py

```diff
@@ -95,7 +95,9 @@ def signal_handler(signum, frame):
             job_id=self.id,
             pool=self.pool,
         ):
-            self.log.info("Task is not able to be run")
+            if self.task_instance.state != State.RUNNING:
+                # Only log about this for non-running tasks.
```

Comment:
How about notifying that the task is running?

Reply:
That's a good point. Done.